netbox/dcim/graphql/types.py

```diff
@@ -120,8 +120,7 @@ class ModularComponentTemplateType(ComponentTemplateType):
 )
 class CableTerminationType(NetBoxObjectType):
 
-    @strawberry_django.field
-    def termination(self) -> List[Annotated[Union[
+    termination: List[Annotated[Union[
```

Comment:
Should this be a List? It returns a single object. (Same question for the other GFK fields.)

Reply:
good catch - yes, shouldn't be list.